plugins/inputs/redis/redis.go

```diff
@@ -189,6 +189,24 @@ func gatherInfoOutput(
 			}
 		}
 
+		if section == "Commandstats" {
+			//#cmdstat_get:calls=63102137,usec=166854619,usec_per_call=2.64
+			tmp := strings.Split(line, ":")
+			tmp_metrics := strings.Split(tmp[1], ",")
```

Comment:
Check for the length of the splits before indexing, just in case the input is bad for some reason.  In the case of not having a ":", call acc.AddError and continue.